tests/acceptance/features/bootstrap/AuthContext.php

```diff
@@ -83,19 +90,26 @@ public function getAppToken() {
 	 * get the app token that was last generated
 	 * app acceptance tests that have their own step code may need to use this
 	 *
-	 * @return string app token
+	 * @return array app token
```

Comment:
```suggestion
	 * @return array app tokens
```
(if you need to edit again anyway, then may as well make this little change)